govuk-notify-spi/src/main/scala/uk/gov/nationalarchives/notifyspi/NotifyEmailSenderProviderFactory.scala

```diff
@@ -0,0 +1,32 @@
+package uk.gov.nationalarchives.notifyspi
+
+import org.keycloak.Config
+import org.keycloak.email.{EmailSenderProvider, EmailSenderProviderFactory}
+import org.keycloak.models.{KeycloakSession, KeycloakSessionFactory}
+import uk.gov.service.notify.NotificationClient
+
+class NotifyEmailSenderProviderFactory extends EmailSenderProviderFactory {
+
+  private val emailSenderId = "govuknotify"
+  private val apiKey = "GOVUK_NOTIFY_API_KEY"
+
+  override def create(session: KeycloakSession): EmailSenderProvider = {
+    val clientApiKey: String = sys.env.get(apiKey) match {
```

Comment:
Is it possible to get environment variables through application.conf, like we do in other projects? Or does that not work here?

Reply:
Been trying for the last few days to get this to work, but it doesn't, as the when running the jar in Keycloak the application.conf it not being found, even when explicitly setting the location of the application.conf